met-web/src/components/feedback/listing.tsx

```diff
@@ -122,12 +140,28 @@ const FeedbackListing = () => {
                     width="100%"
                     justifyContent="flex-end"
                     sx={{ p: 2 }}
-                ></Stack>
+                >
+                    {authorized ? (
```

Comment:
reminder that you can use <When> for this

Reply:
Good suggestion